apps/bfd-pipeline/bfd-pipeline-rda-grpc-apps/README.md

```diff
@@ -157,7 +157,7 @@ export CLASSPATH=$cp
 
 java \
   -Ds3.local=true \
-  gov.cms.bfd.pipeline.rda.grpc.server.RdaServerApp \
+  gov.cms.bfd.pipeline.rda.grpc.apps.RdaServerApp \
```

Comment:
I think we also need to update lines 11, 136, 142 & 148

Reply:
Yep, I missed a commit